docs/Documentation/Resource_Management_System/index.md

```diff
@@ -0,0 +1,55 @@
+# Lex Allocation Management System (Beta)
+*Lex is a system for managing your high-performance computing (HPC) resource assignments, account information, and tracking utilization.*
+
+*Note:  Lex is currently in development with beta functionality available.*
+
+## Access
+
+An [NREL HPC account](https://www.nrel.gov/hpc/user-accounts.html) is required to access Lex. To login to Lex, open a web browser to [https://hpcprojects.nrel.gov](https://hpcprojects.nrel.gov/login/?next=/). Login with your NREL HPC username and password. An OTP token is not required to authenticate. 
```

Comment:
"To log in to Lex..." and "Log in with...". (Oddly, the noun is a 'login', while the verb form is to 'log in'.)